api/user_test.go

```diff
@@ -2339,3 +2339,84 @@ func TestAutocompleteUsers(t *testing.T) {
 		t.Fatal("should have errored - bad team id")
 	}
 }
+
+func TestGetUsername(t *testing.T) {
+	th := Setup()
```

Comment:
I believe if you use `th := Setup().InitBasic()` you create a client, a team, a channels and a user that you can use instead of having to create them all manually